package.json

```diff
@@ -100,7 +100,7 @@
   },
   "devDependencies": {
     "@spinnaker/eslint-plugin": "1.0.7",
-    "@spinnaker/mocks": "1.0.2",
+    "@spinnaker/mocks": "1.0.3",
```

Comment:
in the future, might be cleaner in the commit history to have this package/lockfile bump in a separate PR/commit?

Reply:
yes I am going to make a separate PR and rebase. I did this temporarily so I wouldn't be blocked, then got side-tracked haha.